lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -583,18 +649,6 @@ public function get_create_params() {
 		);
 	}
 
-	/**
-	 * Allow the font face post type to be managed through the REST API.
-	 *
-	 * @since 6.5.0
-	 *
-	 * @param WP_Post_Type|string $post_type Post type name or object.
-	 * @return bool Whether the post type is allowed in REST.
-	 */
-	protected function check_is_post_type_allowed( $post_type ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable -- required by parent class
-		return true;
-	}
-
```

Comment:
This is not needed now that we have `show_in_rest` set to `true` for `wp_font_face`.